pallets/xyk/src/tests.rs

```diff
@@ -300,10 +136,10 @@ fn create_pool_N_already_exists_other_way() {
 #[test]
 fn create_pool_N_not_enough_first_asset() {
     new_test_ext().execute_with(|| {
-        let acc_id: u64 = 2;
+        let accId: u64 = 2;
```

Comment:
looks like you are using old version of the `tests.rs`
please update your branch